### PR TITLE
New actions on disptach

### LINKED ISSUE #64: Rejected button on dispatch
Add theaction buttons allowing to change the statuses within the dispatch statuses

### PR DESCRIPTION
**Pull request by alexisjanvier**
dispatchActions

### CODE REVIEW

File: apps/admin/src/request/dispatcher/DispatchActions.js [hunk 1]
```diff
@@ -60,42 +84,68 @@ const DispatchActions = ({ record }) => {
...
-                >Affecter</Button>
+        )
+    }
+
+    if (record.status === 'DISPATCH_REJECTED') {
+        return (
+            <Grid container className={classes.container}>
+                <Grid item xs={12} justify="flex-end" alignItems="flex-start" container>
+                    <Button variant="outlined" disabled={disabled} onClick={toDispatch}>
+                        Remettre à disptacher
```
Comment (floo51) [inline apps/admin/src/request/dispatcher/DispatchActions.js]:
```suggestion
                        Remettre à dispatcher
```

File: apps/api/src/request/authorization/permissions.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
-const ownProductionManagement = (user, from, to) => {
-    return user.productionManagementIds.includes(from.productionManagementId);
+const ownProductionManagement = (user, from) => {
+    return (
+        user.role === "admin" ||
```
Comment (floo51) [inline apps/api/src/request/authorization/permissions.js:3]: Much better than setting all production managements to the admin profile every time 👍

**Review by floo51 — CHANGES_REQUESTED** [review]

File: apps/admin/src/request/dispatcher/DispatchActions.js [hunk 3]
```diff
@@ -60,42 +84,68 @@ const DispatchActions = ({ record }) => {
...
+        return (
+            <Grid container className={classes.container}>
+                {record.status === 'DISPATCH_TODO' && <Grid item xs={3}>
+                    <Button variant="contained" color="secondary" disabled={disabled} onClick={postpone}>
+                        Mettre en attente
+                    </Button>
+                </Grid>}
+                {record.status === 'DISPATCH_PENDING' && <Grid item xs={3}>
+                    <Button variant="contained" color="secondary" disabled={disabled} onClick={toDispatch}>
+                        A disptacher
```
Comment (floo51) [inline apps/admin/src/request/dispatcher/DispatchActions.js]:
```suggestion
                        A dispatcher
```

**Comment by alexisjanvier** [comment]
@floo51 Review applied

**Comment by Kmaschta** [comment]
J'étais en train de review :(

**Pull request merged**